### PR TITLE
add yarn.lock + refactor dist.sh usage + minor package upds

### PR DESCRIPTION
**Pull request by Kouprin**
There is `dist.sh` in the root and symlinks in the directories.

### CODE REVIEW

**Comment by Kouprin** [comment]
@ailisp 
```
[2020-08-27T12:13:38Z] HEAD detached at 5e6bb03
--
  | [2020-08-27T12:13:38Z] Changes not staged for commit:
  | [2020-08-27T12:13:38Z]   (use "git add <file>..." to update what will be committed)
  | [2020-08-27T12:13:38Z]   (use "git restore <file>..." to discard changes in working directory)
  | [2020-08-27T12:13:38Z] 	modified:   nearbridge/yarn.lock
  | [2020-08-27T12:13:38Z] 	modified:   nearprover/yarn.lock
  | [2020-08-27T12:13:38Z] 	modified:   token-locker/yarn.lock
```
What's wrong with `git status` at this point? I cannot reproduce it locally.

**Comment by ailisp** [comment]
> What's wrong with git status at this point? I cannot reproduce it locally.

I guess the reason is ci runner has a different version of yarn, and they don't produce same formalized format, such as this:

https://github.com/near/rainbow-bridge-sol/pull/7/files#diff-3f455b1268ce34789f4d4568f1de0c34R2568

Try adding a `yarn --version` and `git diff` in the step, and see what was different? Also we don't have to debug this, what we want to test is "compile all solidity contracts on ci produce same result as compiled contracts in the commit", so change it to no ".sol", ".bin", ".abi" files are changed in git diff?

File: nearbridge/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
         "eslint-plugin-standard": "^4.0.0",
         "eth-gas-reporter": "^0.2.11",
         "ganache-cli": "^6.9.1",
-        "rainbow-bridge-lib": "1.0.0",
+        "rainbow-bridge-lib": "^1.0.0",
```
Comment (ailisp) [inline nearbridge/package.json:28]: Should we pin a rainbow-bridge-lib version here?

**Review by ailisp — APPROVED** [review]

File: nearbridge/package.json (hunk 1 above)
Comment (Kouprin) [reply nearbridge/package.json:28]: I suggest to use the latest version if possible. It helps us to run the latest version in CI and distribute the changes between components quickly.
Do you think it's necessary to pin?

Comment (ailisp) [reply nearbridge/package.json:28]: Sounds good! I think here it's good to use latest rainbow-bridge-lib and in rainbow-bridge-cli we should pin lib/sol/rs to ensure cli deterministic

**Pull request merged**